internal/provisioner/cluster_installation_provisioner.go

```diff
@@ -1081,6 +1081,7 @@ func clusterInstallationBaseLabels(installation *model.Installation, clusterInst
 	return map[string]string{
 		"installation-id":         installation.ID,
 		"cluster-installation-id": clusterInstallation.ID,
+		"dns":                     strings.TrimSuffix(clusterInstallation.ClusterID, "-public"),
```

Comment:
ClusterIDs should only ever be our standard 26 char IDs. Is there a reason we expect this suffix to exist?

Reply:
@gabrieljackson Added this suffix mostly for sanity reasons to know that these ingreses will be handled by our public external DNS we will have a different one for the private DNSes we are using but for this we don't need any provisioner code changes as it affects only argo utilities